test/emitters/delegate_test.rb

```diff
@@ -5,6 +5,13 @@ def test_format_zone
     assert_equal 'UTC', e.format(Time.now.utc)
   end
 
+  def test_format_zone_current
+    e = Stamp::Emitters::Delegate::ZONE
+
+    assert_equal 'MST', e.format(Time.current.in_time_zone('MST'))
+  rescue NoMethodError
```

Comment:
if forcinf active support, then can remove this.
if not, maybe a verbal indicator that this is skipped?